modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -88,7 +88,24 @@ class PackageDeclarationNode extends React.Component {
                 packageNameEditing: false,
             });
         }
-        this.props.model.setPackageName(this.state.packageDefValue);
+        if (this.state.packageDefValue) {
+            const pkgName = 'package ' + this.state.packageDefValue + ';';
```

Comment:
Best to use string templates here.

Reply:
Fixed with commit 03ae4b262b1b3d7bb6b90f4060106e4e4fae1334